etc/kayobe/kolla.yml

```diff
@@ -181,6 +181,13 @@ kolla_build_blocks:
                -e '/#baseurl.*/a baseurl={{ repo.url }}' /etc/yum.repos.d/{{ repo.file }}{% if not loop.last %} &&{% endif %} \
     {% endfor %}
 
+  cloudkitty_base_footer: |
+    # Backport of NOTNUMBOOL: https://review.opendev.org/c/openstack/cloudkitty/+/774084
+    # Support customising Prometheus queries: https://review.opendev.org/c/openstack/cloudkitty/+/817275
+    RUN pip install git+https://github.com/stackhpc/cloudkitty.git@stackhpc/wallaby
+  horizon_footer: |
+    # Fixes CloudKitty pricing panel: https://github.com/stackhpc/cloudkitty-dashboard/pull/1
+    RUN pip install git+https://github.com/stackhpc/cloudkitty-dashboard.git@stackhpc/wallaby
```

Comment:
Could you use the kolla_sources approach rather than footers?

Reply:
Oh yeah, missed that thanks.